### DISCUSSION TITLE
How can we add media query and hover style using grapesjs component model

### DISCUSSION THREAD

**Discussion by dev4322** [GrapesJS/grapesjs#3771] [Q&A]
we can add style in grapesjs model like this.
 model.addStyle({
      'background-color': 'red',
});

can anyone tell me how can we add media rules or hover rules using grapesjs model.
I know about CssComposer. Is there any way of adding media query style using model.

**Comment by artf** [comment]
Check this section: https://grapesjs.com/docs/modules/Components.html#components-css
